services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +43,62 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
+  if(!event){
+    return ERR_CODE_INVALID_ARG;
+  }
+
+
+  if (!thermalMgrQueueHandle) {
+    return ERR_CODE_INVALID_QUEUE_MSG;
```

Comment:
this error is if we have not created the queue yet so it should be `ERR_CODE_INVALID_STATE` not invalid queue msg